scripts/handlers.js

```diff
@@ -0,0 +1,105 @@
+import * as elements from './elements.js';
+import {
+  getTotalOfButtonsAndRemoveActiveClass,
+  loadTestimonials,
+} from './utils.js';
+
+export function handleMobileBtnClick() {
+  elements.mobileNav.classList.toggle('close');
+}
+
+export function handleTeamMemberEnter(event) {
+  const links = event.currentTarget.querySelector('.team-member__links');
+  links.classList.add('active-element');
+}
+
+export function handleTeamMemberOut(event) {
+  const links = event.currentTarget.querySelector('.team-member__links');
+  links.classList.remove('active-element');
+}
+
+export function handlePortafolioImageEnter(event) {
+  const imageInfo = event.currentTarget.querySelector('.portafolio__info');
+  imageInfo.classList.add('active-element');
+}
+
+export function handlePortafolioImageOut(event) {
+  const imageInfo = event.currentTarget.querySelector('.portafolio__info');
+  imageInfo.classList.remove('active-element');
+}
+
+export function handleQuestionClick(event) {
+  const clickedQuestion = event.currentTarget;
+  const clickedAnswer = clickedQuestion.querySelector('.question__answer');
+  if (clickedAnswer.classList.contains('open-answer')) {
+    return;
+  }
+  elements.questions.forEach(otherQuestion => {
+    if (otherQuestion !== clickedQuestion) {
```

Comment:
when comparing references the object can have everything you need and change the reference because another render changed the position in the memory, that's why this is a bad practice

Reply:
Done